final_prep/README.md

```diff
@@ -23,52 +23,57 @@ Some things to consider as you refactor include...
+- [x] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+- ![4thweekmarchmod1](https://user-images.githubusercontent.com/99222755/156617578-c59b8a3b-efd8-45c9-b651-9821eaf83517.PNG)
+
 - `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- ![3rdweekMarchmod1](https://user-images.githubusercontent.com/99222755/156617713-55c50f94-b083-403f-9085-81dbbb656151.PNG)
 
+- `Add Week 3 Screenshot Here`
+- ![Aprilweek1mod1](https://user-images.githubusercontent.com/99222755/156617779-ae92e4ec-8081-40eb-9d65-a77c8d821216.PNG)
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
-- [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [X] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: [DTR Gist Link](https://gist.github.com/SarahEHall/29d16752ef81e560f41ac1f82fb43106)
 
 ## Lesson Prep
 
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [x] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [x] Add a link to your gist here: [Pre-teaching Gist](https://gist.github.com/SarahEHall/7a10c84d910a7f5b70fcd82ab8aa01be)
 
 ## Group Work Prep
 
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [x] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+  - [x] Add a link to your gist here: [DTR guiding questions gist](https://gist.github.com/SarahEHall/5ced41269683ecd07fce892c6b38604b) 
```

Comment:
You seem so thoughtful and self aware; I think you'll be an awesome colleague and teammate!